src/addEditRemove.js

```diff
@@ -0,0 +1,61 @@
+import { save, load } from './data';
+
+const theBigList = document.querySelector('.lists');
+
+export function ShowAll(element) {
```

Comment:
#1 
Hey @kelomo2502 👋 , you made a solid effort developing this project, good work 👏 . Though in order to adhere to ES6 syntax requirements, you should declare your functions in the format **'const showAll = (element) => {};**' rather than **' function showAll(element){};'**.Remember to always use camelcase when naming functions, variables and constants. Cheers! 😄